components/Cases/CaseLink.tsx

```diff
@@ -33,7 +33,7 @@ const getLink = (
     case 'API_WarningNote':
       return `/people/${caseFormData.mosaic_id}/warning-notes/${
         (caseFormData as WarningNoteCaseFormData).warning_note_id
-      }`;
+      }/view`;
```

Comment:
i would add a test case that checks this url is spat out as you expect when you feed it warning note data

Reply:
Done in commit: 47efd66